Adyen/HttpClient/HttpURLConnectionClient.cs

```diff
@@ -160,7 +160,6 @@ public string Post(string endpoint, Dictionary<string, string> postParameters, C
             {
                 httpWebRequest.Proxy = config.Proxy;
             }
-            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Is this content from another PR bleeding into this one?

Reply:
Yes I saw this one [here](https://github.com/Adyen/adyen-dotnet-api-library/compare/develop...PW-7372#diff-8b602ca15935ec373df692637c85201512137f432ab908df44fa25ad74e8467bR49)

Reply:
it is removed in another pr